solutions/CSES Pizzeria Queries.mdx

```diff
@@ -0,0 +1,82 @@
+---
+id: cses-2206
+source: CSES
+title: Pizzeria Queries
+author: Dong Liu
+---
+
+# Solution
+
+When we query the minimum cost to buy a pizza at point $i$, we can split it into two cases ($i$$\rightarrow$$j$)
+1. going downwards ($j$<$i$) : $cost$ would be $p_j-j+i$
+2. going upwards ($j$>$i$) : $cost$ would be $p_j+j-i$
+
+Notice that we can caculate the $+i$ or $-i$ after the query since it is constant.
```

Comment:
```suggestion
Notice that we can calculate the $+i$ or $-i$ after the query since it is constant.
```
This isn't really clear. Maybe something like "Since $+i$ and $-i$ are constant, we can calculate the best cost without them and then add them" would be better?